### DISCUSSION TITLE
Как Zapret натравить только на два устройства в сети?

### DISCUSSION THREAD

**Discussion by almirus** [bol-van/zapret#395] [Q&A]
Добрый день! Так как для обхода замедления youtube для Android TV использую конфиг c wssize 
```
MODE=nfqws
MODE_HTTP=1
MODE_HTTP_KEEPALIVE=0
MODE_HTTPS=1
MODE_QUIC=1
MODE_FILTER=hostlist

NFQWS_OPT_DESYNC="--dpi-desync=disorder2 --dpi-desync-ttl=4 --dpi-desync-ttl6=2 --dpi-desync-split-pos=1 --wssize 1:6 --dpi-desync-fooling=md5sig" 
NFQWS_OPT_DESYNC_QUIC="--dpi-desync=disorder2 --dpi-desync-repeats=6"
```

Хочется подвергать дурению только трафик с двух тв (у них статик IP) и не трогать трафик с других устройств.
Zapret сейчас крутится на OpenWRT

**Comment by bol-van** [comment]
Надо фиксировать IP и лезть в ip/nf tables.
Как вариант - через custom script. Либо руками.
